tests/Doctrine/Tests/Common/DataFixtures/Purger/ORMPurgerTest.php

```diff
@@ -54,4 +55,18 @@ public function testTableNameWithSchema()
         $tableName = $method->invokeArgs($purger, [$metadata, $platform]);
         $this->assertStringStartsWith('test_schema', $tableName);
     }
+
+    public function testGetTableNameQuoted() : void
+    {
+        $em       = $this->getMockAnnotationReaderEntityManager();
+        $metadata = $em->getClassMetadata(self::TEST_ENTITY_GROUP);
+        $platform = $em->getConnection()->getDatabasePlatform();
+        $purger   = new ORMPurger($em);
+        $class    = new ReflectionClass(ORMPurger::class);
+        $method   = $class->getMethod('getTableName');
+        $method->setAccessible(true);
+        $tableName = $method->invokeArgs($purger, [$metadata, $platform]);
+        $this->assertStringStartsWith('"', $tableName);
```

Comment:
Why not asserting the full table name?

Reply:
I think I was focused on ensuring the quotes were there without having to worry about the case of the table name itself. As the table name isn't given in the entity, the function needs to work regardless of whether the table name is 'Group', or 'GROUP', or 'Group', etc. and this will prove that.

That said, the test is obviously run in a minimal config environment so the test entity's table name should default to 'Group'.